src/main/g8/README.md

```diff
@@ -1,8 +1,10 @@
-## Template sbt project that is cross compiled using Dotty and Scala 2
+## sbt project cross-compiled with Dotty and Scala 2
 
 ### Usage
 
-This is a normal sbt project, you can compile code with `sbt '+ compile'` and run it
```

Comment:
When people compile/run their projects, most of the time they don't want to cross-compile, they just want to run with the main scalaVersion. This is what the instructions now say. I also added a link to the cross-compilation documentation of sbt so people know what to do when they want to cross-compile.